ang/crmCaseType.js

```diff
@@ -433,8 +433,10 @@
         } else {
           CRM.loadForm(CRM.url('civicrm/admin/reltype', {action: 'add', reset: 1, label_a_b: roleName, label_b_a: roleName}))
             .on('crmFormSuccess', function(e, data) {
-              roles.push({name: data.relationshipType[REL_TYPE_CNAME]});
-              $scope.relationshipTypeOptions.push({id: data.relationshipType[REL_TYPE_CNAME], text: data.relationshipType.label_b_a});
+              var newTypes = _.values(data.relationshipType);
+              var newType = newTypes[0];
```

Comment:
Thanks for reviewing. I can update that. By the way if you have any idea how to write a test for this I've been struggling with that.

Reply:
This is probably a tricky thing to test since it uses a non-angular form via `CRM.loadForm`. I haven't found any existing tests for other instances of that pattern.

The best one could do without opening that can of worms would be to refactor the `crmFormSuccess` callback into a separate function and call that with test data, but then you're not really testing against regressions in `crmFormSuccess`, so arguably there's not much point in doing so.